js/PuzzleOne.js

```diff
@@ -0,0 +1,354 @@
+// TEST COMMENT COMMIT
+// Puzzle navigation Buttons in Nav bar
+const NAV1 = document.getElementById("puzzle1-nav-button");
+const NAV2 = document.getElementById("puzzle2-nav-button");
+const NAV3 = document.getElementById("puzzle3-nav-button");
+const NAV4 = document.getElementById("puzzle4-nav-button");
+const NAV5 = document.getElementById("puzzle5-nav-button");
+
+// Puzzle Elements
+const PE1 = document.getElementById("puzzle1-element");
+const PE2 = document.getElementById("puzzle2-element");
+const PE3 = document.getElementById("puzzle3-element");
+const PE4 = document.getElementById("puzzle4-element");
+const PE5 = document.getElementById("puzzle5-element");
+
+// Dialogue Elements
+const DIA0 = document.getElementById("dialogue-0");
+const DIA1 = document.getElementById("dialogue-1");
+const DIA2 = document.getElementById("dialogue-2");
+const DIA3 = document.getElementById("dialogue-3");
+const DIA4 = document.getElementById("dialogue-4");
+const DIA5 = document.getElementById("dialogue-5");
+const DIA6 = document.getElementById("dialogue-6");
+
+// Welcome Screen elements
+const PE0 = document.getElementById("puzzle0-element");
+const P0Start = document.getElementById("p0-p1-navbutton");
+
+// ANSWER TO PUZZLE 1
+const P1A = "map";
+
+// ANSWER TO PUZZLE 2
+const P2A = "20";
+
+// ANSWER TO PUZZLE 3
+const P3A = "patchyass";
+
+// ANSWER TO PUZZLE 4
+const P4A = "chenga";
```

Comment:
Neke promenljive (poput ove) koristis samo jednom. Pogledaj sta je `scope` u JavaScriptu i vidi kako bi mogao to da iskoristis.

Reply:
> Neke promenljive (poput ove) koristis samo jednom. Pogledaj sta je `scope` u JavaScriptu i vidi kako bi mogao to da iskoristis.

Aaa ok shvatam sad kad sam pogledao, ima smisla.

Mada mislim da kada sam kucao kod nisam baš imao u vidu šta će biti puzle i njeni odgovori, već sam ih samo deklarisao na početku gde su vidljivi da bi lakše testirao kod. 

U suštini nisam čak ni morao da koristim const za odgovore, već samo tokom `if` statement da stavim condition `(a1.value == "patchyass") {}`